### PR TITLE
AsciinemaWidget

### PR DESCRIPTION
**Pull request by farzadab**
The widget code is from https://consolelogdupa.com/embedding-asciinema-asciicast-in-your-docusaurus-2-docs
I have not modified it.

### CODE REVIEW

File: packages/docs/docs/about.mdx [hunk 1]
```diff
@@ -88,4 +88,11 @@ function CharacterGenerator() {
 }
 ```
 
-[![AI.JSX demo](https://asciinema.org/a/kLwqpHVf2jsOU0vxEpl8f47zw.svg)](https://asciinema.org/a/kLwqpHVf2jsOU0vxEpl8f47zw)
+import AsciinemaWidget from '../src/components/AsciinemaWidget';
```
Comment (farzadab) [inline packages/docs/docs/about.mdx:91]: Not sure if I should put this at the top or not.
Probably this is easier to copy&paste later on.

**Comment by mdwelsh** [comment]
Can you please set the "poster" value to "npt:0:10" so the thumbnail shows the text?

https://github.com/asciinema/asciinema-player#poster

File: packages/docs/docs/about.mdx (hunk 1 above)
Comment (mdwelsh) [reply packages/docs/docs/about.mdx:91]: This is fine.

**Review by mdwelsh — APPROVED** [review]

**Comment by farzadab** [comment]
Done. Setting to auto-merge.

**Pull request merged**